src/platforms/android/configuration/integrations/apollo3.mdx

```diff
@@ -10,14 +10,16 @@ categories:
 
 <Note>
 
-Capturing transactions requires that you first <PlatformLink to="/performance/">set up performance monitoring</PlatformLink> if you haven't already.
+To be able to capture transactions, you'll need to first <PlatformLink to="/performance/">set up performance monitoring</PlatformLink>.
```

Comment:
This sentence is in a bunch of files, mind updating all of them? just search for:

> Capturing transactions requires that you first <PlatformLink to="/performance/">set up performance monitoring</PlatformLink> if you haven't already.

Reply:
Will do!